osu.Game/Graphics/UserInterface/OsuSliderBar.cs

```diff
@@ -203,28 +203,22 @@ private void playSample(T value)
             channel.Play();
         }
 
-        private void updateTooltipText(T value)
+        protected virtual LocalisableString GetTooltipText(T value)
```

Comment:
Keep in mind this now results in two different ways to customize the tooltip text, one by overriding `TooltipText` (which is the current way to customize tooltip texts) and now another way by overriding `GetTooltipText`.

I think the new `GetTooltipText` method is a better way and all current overrides of `TooltipText` should migrate to it, and no longer make `TooltipText` overridable.

Reply:
Right, I was looking for exactly how it was done and couldn't find it. I thought `TooltipText` had to be `set` to work but I guess not. Will wait for more opinions on how to take this PR then, since we could migrate to using that and revert the flow changes I've made here.

Reply:
should i update all other usages here?

Reply:
I've applied the changes here, seems pretty good.